networking/synthetic/scripts/tuning.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+. /usr/lib/tuned/functions
+
+start() {
+    tuna -c22 -q p5p1 -m -x
+    tuned-adm profile stac || exit 1
+    # additional tuning necessary if using nohz_full:
+    if grep -q nohz_full /proc/cmdline
+    	then
+    	for i in `pgrep rcu` ; do taskset -pc 0 $i ; done
```

Comment:
just so you know, this puts all rcu tasks on core 0.  its fine, but i think in reality that should be not only core 0 but all housekeeping cores (maybe a customer has cores 0,12 as their housekeeping cores.  no need to change this now -- just wanted to point it out.